gp2gp-translator/src/integrationTest/java/uk/nhs/adaptors/pss/translator/LargeMessagingIT.java

```diff
@@ -156,8 +170,39 @@ private void sendInboundMessageToQueue(String json) {
             .replace(CONVERSATION_ID_PLACEHOLDER, getConversationId());
         getMhsJmsTemplate().send(session -> session.createTextMessage(jsonMessage));
     }
+
     private boolean hasContinueMessageBeenReceived() {
         var migrationStatusLog = getMigrationStatusLogService().getLatestMigrationStatusLog(getConversationId());
         return CONTINUE_REQUEST_ACCEPTED.equals(migrationStatusLog.getMigrationStatus());
     }
+
+    private ArrayList<String> sendInboundMessageToQueueAndExtractMids(String filename) {
+        var jsonMessage = readResourceAsString(filename)
+            .replace(NHS_NUMBER_PLACEHOLDER, getPatientNhsNumber())
+            .replace(CONVERSATION_ID_PLACEHOLDER, getConversationId());
+
+        getMhsJmsTemplate().send(session -> session.createTextMessage(jsonMessage));
```

Comment:
Minor: Could have `sendInboundMessageToQueue` return the `jsonMessage`

Reply:
yeah I it would have meant having to change all the other tests, as it's used in most of the tests and not expecting anything to be returned